README.md

```diff
@@ -24,6 +24,65 @@ image. On Fedora this is the `fedora-toolbox` image. This image is used to
 create a toolbox container that seamlessly integrates with the rest of the
 operating system.
 
+## Goals and Use Cases
```

Comment:
How about putting the `Goals and Use Cases` section below `Usage`? Just so that first-time readers and newcomers get a quick practical introduction before getting into the more nitty-gritty details.

Not a very strong opinion, though. :)

Reply:
yeah I thought about that, but it seemed like we are jumping right to how to use it without explaining what it is for first. I'm open to moving it.

@cgwalters @matthiasclasen - any opinion here?

Reply:
the suggested change is fine with me

Reply:
ok I moved it to after `Usage`!